.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
```

Comment:
- It's important to ensure that the replacement patterns in the sed commands are accurate and cover all possible cases to avoid unintended modifications.